web-access-control/protected-operation/write-access-agent.feature

```diff
@@ -153,5 +154,4 @@ Feature: Only authenticated agents can write (and only that) a resource when gra
       | Public | cannot | DELETE | fictive   | WAC       | inherited | [401]                |
       | Public | cannot | DELETE | rdf       | no        | WAC       | [401]                |
       | Public | cannot | DELETE | rdf       | WAC       | inherited | [401]                |
-      | Public | cannot | DELETE | container | no        | WAC       | [401]                |
-      | Public | cannot | DELETE | container | WAC       | inherited | [401]                |
+      | Public | cannot | DELETE | container | no        | WAC       | [401]
```

Comment:
This is not an expected change, can you revert this one please

Reply:
Curious. I'm not sure where that change came from. It has been reverted.